libraries/cloudharness-common/cloudharness/utils/config.py

```diff
@@ -0,0 +1,49 @@
+import yaml
+
+ALLVALUES_PATH = '/opt/cloudharness/resources/allvalues.yaml'
+
+class _obj(object):
```

Comment:
Class should be uppercase (PEP8). A more informative name also would help read the code (same below). As for the package private, I prefer the convention to importwhat it considered api in the main package __init__ file rather than these clunky _ names

Reply:
the _ names indicate that these classes are "private"
about class name, will change it

Reply:
I understand these are package private (see comment above), I just don't see any reason to make these private. CloudHarnessConfig surely shouldn't be as its interface is important to be known outside and may be used in type hinting. So my suggestion is to use the export at the module top level to make the module api explicit rather than make everything private.

Reply:
done in my latest commit